packages/amplication-server/src/core/module/module.service.ts

```diff
@@ -61,13 +64,55 @@ export class ModuleService extends BlockTypeService<
     }
   }
 
-  async create(args: CreateModuleArgs, user: User): Promise<Module> {
+  async findMany(args: FindManyModuleArgs, user?: User): Promise<Module[]> {
+    const prismaArgs = {
+      ...args,
+      where: {
+        ...args.where,
+      },
+    };
+
+    if (user) {
+      const subscription = await this.billingService.getSubscription(
+        user.workspace?.id
+      );
+
+      await this.analytics.trackWithContext({
+        properties: {
+          planType: subscription.subscriptionPlan,
+        },
+        event: EnumEventType.SearchAPIs,
+      });
+    }
+
+    return super.findMany(prismaArgs);
+  }
+
+  async create(
+    args: CreateModuleArgs,
+    user: User,
+    trackEvent = true
```

Comment:
should it default to true or false?

Reply:
true, sending false from the inner function